src/main/java/healthy/lifestyle/backend/users/service/UserServiceImpl.java

```diff
@@ -192,4 +188,8 @@ public UserResponseDto getUserDetailsById(long userId) {
 
         return modelMapper.map(userOptional.get(), UserResponseDto.class);
     }
+
+    private boolean notEmptyUserData(String input) {
```

Comment:
The name of this method doesn't reflect user data. Maybe, rename it to `nonEmptyString`, or maybe there is some default method for checking whether String is empty?